packages/react-devtools-shared/src/hook.js

```diff
@@ -21,6 +26,15 @@ export function installHook(target: any): DevToolsHook | null {
     return null;
   }
 
+  // Keep in sync with src/backend/console.js
+  const OVERRIDE_CONSOLE_METHODS = ['error', 'trace', 'warn', 'log'];
+  const DARK_MODE_DIMMED_WARNING_COLOR = 'rgba(250, 180, 50, 0.5)';
+  const DARK_MODE_DIMMED_ERROR_COLOR = 'rgba(250, 123, 130, 0.5)';
+  const DARK_MODE_DIMMED_LOG_COLOR = 'rgba(125, 125, 125, 0.5)';
+  const LIGHT_MODE_DIMMED_WARNING_COLOR = 'rgba(250, 180, 50, 0.75)';
+  const LIGHT_MODE_DIMMED_ERROR_COLOR = 'rgba(250, 123, 130, 0.75)';
+  const LIGHT_MODE_DIMMED_LOG_COLOR = 'rgba(125, 125, 125, 0.75)';
```

Comment:
Idea: We could use constants for these variables, and inject them via the Webpack config (like we do for `__EXTENSION__` and `process.env.DEVTOOLS_PACKAGE`)